core/src/sidebar/sidebar.module.css

```diff
@@ -0,0 +1,33 @@
+.username {
+	font-weight: 600;
+	font-size: 18px;
+	line-height: 24px;
+	color: var(--gray-8);
+	align-self: stretch;
+}
+
+.role {
+	font-size: 12px;
+	line-height: 18px;
+	color: var(--gray-3);
+	text-transform: capitalize;
+	align-self: stretch;
+}
+
+.header {
+	padding: 24px 0;
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+}
+
+.header:hover,
+.header:active {
+	cursor: pointer;
+}
+
+.picture {
+	width: 32px;
+	height: 32px;
+	margin: 4px 8px 0 12px;
```

Comment:
Don't use margin properties for the direction where are nothing or to align the item.

1. The outer spacing will be made by the `padding` property of the parent.
2. This image already centered when you use `align-items: center`, so what margin-top for?